### ISSUE TITLE
Re: [DVR] Saturday Spin

### ISSUE THREAD

**Issue by Lancea12** [Lancea12/test#433]
From: grant.mandsager@gmail.com<br>
Subject: Re: [DVR] Saturday Spin<br>
First Attachment: 20150322_100637.jpghttps://locker.ifttt.com/f/fc40d6b2-e58f-4fe8-acdb-983a8ca176d4.jpg<br>
Date: March 22, 2015 at 07:32PM<br><br>

My 10am.  In development team news, Ryder and Sydney have probably spent 5 hours on the bike over the last 3 days. World Tour or bust. #earndaddyslove

On Mar 22, 2015 2:23 PM, "Patrick Dolan"  wrote:
It's catchy.On Sunday, March 22, 2015 at 1:27:55 PM UTC-4, Tony Abate wrote:
If I ever open a pub, I shall call it The Dolan Red Brick Wonder Zone. On Sunday, March 22, 2015, Patrick Dolan  wrote:

This is a fair question, and I will try to give it a fair answer.   My garden style U shaped building holds a mix of families and singles across ethnic and cultural boundaries.  It's usually pretty peaceful but it can get randomly noisy and if in the courtyard, or from an upper apartment with windows open, it echoes.  My artist neighbor will hear across the way and one floor above my upstairs neighbor, if he gets a little rowdy on a Sat. night with windows open.

So, Saturday at 3AM my artist neighbor was drunk and singing at the top of his voice with his earplugs in his phone, wasted, while smoking a cig out in the courtyard.  I went to talk to him and he went inside.  Five minutes later cops came by.

This morning, it was either one of the families mobilizing their two kids to do something for the day, OR, it was my single neighbor across the hall coming back from an all nighter with her boyfriend and chilling out there.   Hard to tell, and I was too tired to look.

I went to sleep last night to the sound of what I'm pretty sure is some sort of modern Buddhist monk hip hop.   The family across the way has parties for their kids and a relation is such a monk.

Basically, it's spring in the Dolan red brick wonder zone.  The upstairs neighbor uncharacteristically had to get up at around 6AM to rush out somewhere, so that stirred me up further.

On Sunday, March 22, 2015 at 8:36:00 AM UTC-4, Tom Jawetz wrote:

What were they doing in the courtyard?

<br><br>
via Gmail
